docs/intro/tutorial.rst

```diff
@@ -18,11 +18,12 @@ This tutorial will walk you through these tasks:
 4. Changing spider to recursively follow links
 5. Using spider arguments
 
-Scrapy is written in Python_. If you're new to the language you might want to
-start by getting an idea of what the language is like, to get the most out of
+Scrapy is written in Python_. If you're new to the language, you might want to
+start by getting an idea of what the language is like to get the most out of
```

Comment:
Here, the added comma seems unnecessary to me, while the removed one felt necessary.

Reply:
I just run it though a punctiation checker and it said there's meant to be a comma after "language". But you're right it's not a big deal.

Reply:
It also highlighted the comma after "like" as a mistake